features/video-optimization/automatic-video-format-conversion.md

```diff
@@ -12,10 +12,17 @@ VP9 is an open and royalty-free video coding format developed by Google. It prov
 
```

Comment:
> VP9 is an open and royalty-free video coding format developed by Google. It provides excellent compression and is supported by major browsers.

@abhinavr4 Should we add a similar text for AV1?

Reply:
Added